content/en/tests/flaky_test_management/auto_test_retries.md

```diff
@@ -27,11 +27,45 @@ Ensure [Test Optimization][1] is configured for your test runs.
 
 ### Compatibility
```

Comment:
In preview there's some blank space between the "compatibility" and "configuration" sections that doesn't look very nice. Can we, perhaps, try moving the two headers inside the tabs

Reply:
Oh yeah, you're right. That... looks meh. I'll give it a shot.